src/NuGetTrends.Data/NuGetTrends.Data.csproj

```diff
@@ -11,6 +11,7 @@
 
   <ItemGroup>
     <PackageReference Include="Npgsql.EntityFrameworkCore.PostgreSQL" Version="2.2.4" />
+    <PackageReference Include="Microsoft.EntityFrameworkCore.Design" Version="2.2.6" />
```

Comment:
need to fix breaking changes to go to 3.0+ but even on 2.2 it's failing to run.

Reply:
what's the issue? I remember running during the last PR (where we upgraded to 3.0) and it seemed to work. But not 100% sure anymore. Anyway moving to 3.1 is probably the best thing.

Reply:
Last error I had was: `Unable to create an object of type 'NuGetTrendsContext'. For the different patterns supported at design time, see https://go.microsoft.com/fwlink/?linkid=851728`